Library/Dungeon Fantasy RPG/Dungeon Fantasy RPG Traits.adq

```diff
@@ -2080,13 +2080,8 @@
 				},
 				{
 					"type": "conditional_modifier",
-					"situation": "on all IQ rolls to wake up or to recover from surprise or mental stun",
+					"situation": "on all IQ rolls to wake up and to recover from mental stun",
```

Comment:
Are you sure 'and' is correct here? To my programmer brain, that seems to indicate only rolls for which you are both waking up and recovering from mental stun, not one or the other.

Does DFRPG not have surprise?

Reply:
Ah, yeah, "or" makes sense. There is surprise, but it just causes mental stun, so it's redundant to mention it. (I like to keep the note text as short as possible so it doesn't add too many lines to my printed sheets. Maybe that's just me.)